### PR TITLE
Modified README.md

### PR DESCRIPTION
**Pull request by Gusted**
- Some improvements to the README file.

**Note:** It's handy to use the squash function instead of merge so all Commits will become 1 instead of all commits(with sometimes no mean in the commit name).

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
+const options = {
+  entry: './build/options.js',
+  output: {
+    filename: './main.js',
+    path: __dirname + '/dist/options'
+  },
+  mode: 'production',
+}
+
+module.exports = [inject, background, options];
```
Comment (Gusted) [inline webpack.config.js:28]: I only removed a extra newline that wasn't needed.

**Comment by WillHayCode** [comment]
Yeah I meant to squash last time, whoooops. Good stuff, as always!

**Comment by WillHayCode** [comment]
@Gusted - Would you be interested in being given direct access to the repository, or would you prefer to operate through pull requests? There wouldn't be any expectations or obligations attached with either, more just if there's ever further improvements you have I'm comfortable with them going directly into the repository.

**Comment by Gusted** [comment]
> @Gusted - Would you be interested in being given direct access to the repository, or would you prefer to operate through pull requests?

I must say creating this forks and than pulling the `upstream-master` is well the top of my GIT knowledge. So it will be appreciated.

**Comment by WillHayCode** [comment]
Added!

**Pull request merged**